### DISCUSSION TITLE
images /storage/ not working

### DISCUSSION THREAD

**Discussion by jouniikaheimo** [laravel/framework#34385] [General]
issue get images /storage/

<img width="1517" alt="image" src="https://user-images.githubusercontent.com/32259223/93458050-79ee8f80-f8e8-11ea-93b0-40a82a507cc1.png">

and xdebug 

<img width="1473" alt="image" src="https://user-images.githubusercontent.com/32259223/93458159-a2768980-f8e8-11ea-8201-fabb8858e677.png">

and project folders

<img width="988" alt="image" src="https://user-images.githubusercontent.com/32259223/93459334-3bf26b00-f8ea-11ea-935a-5951f61ed6cf.png">

result in firefox 

<img width="1612" alt="image" src="https://user-images.githubusercontent.com/32259223/93458404-ee293300-f8e8-11ea-97c2-113a20e5a1cf.png">

and lasted route to controller asset() function

Route::get('/tenancy/assets/{path?}', 'TenantAssetsController@asset')
    ->where('path', '(.*)')
    ->name('tenancy.asset');

i cant understant where is issue ?

**Comment by kangzoel** [comment]
have you run the ```php artisan storage:link```?

**Reply by jouniikaheimo** [reply]
No because i want use images,pdf etc outside public folder

**Reply by kangzoel** [reply]
Have you tried the native way? Does it work?

**Reply by jouniikaheimo** [reply]
What you mean native way ?

**Reply by kangzoel** [reply]
Here's the example of opening an image: https://www.php.net/manual/en/function.fpassthru.php

**Reply by jouniikaheimo** [reply]
https://stackoverflow.com/questions/21133712/laravel-load-images-stored-outside-public-folder i try this because someone same issue but nothing Works.. i try tomorrow that native way :)
